README.md

```diff
@@ -1,74 +1,180 @@
-# scaleway-k8s-node-coffee
+<p align="center"><img width="50%" src="medias/cli-artwork.png" /></p>
 
-The Coffee Machine for Scaleway Kubernetes Nodes
+# scaleway-k8s-node-coffee ☕
 
-## Whats is it?
+Kubernetes toolkit controller project for Scaleway k8s nodes. Initially from [scaleway-k8s-node-coffee](https://github.com/Sh4d1/scaleway-k8s-node-coffee)
```

Comment:
Maybe change the phrasing, as it is `scaleway-k8s-node-coffee` :smile:

Reply:
My bad, I forgot to edit this when I copy/pasted from my local version :grimacing: